src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/UnzipperTest.java

```diff
@@ -0,0 +1,32 @@
+package ch.admin.bar.siard2.cmd.utils.siard;
+
+import ch.admin.bar.siard2.cmd.utils.SiardProjectExamples;
+import ch.admin.bar.siard2.cmd.utils.TestResourcesResolver;
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.IOException;
+
+public class UnzipperTest {
+
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    @Test
+    public void unzip_expectUnzippedArchive() throws IOException {
```

Comment:
``
```suggestion
    public void shouldUnzipArchive() throws IOException {
```

Reply:
Thanks for your comment, @StephGit. I prefer using the 'calledMethod_contextInformation_expectedResult' pattern for unit tests. If there isn't a specific convention that requires a different pattern, I would like to stick with my current solution. I hope that's okay with you.